### ISSUE TITLE
Do we really need to wrap a function with a class and a static method?

### ISSUE THREAD

**Issue by jgomezdans** [multiply-org/multiply-core#10]
In reproject, the function is wrapped around in a class and is decorated with a static method. In Python, you'd just put the function definition without the class boiler plate, although this might not be the convention in other languages (Java, c#). Is there a specific reason for this?

**Comment by TonioF** [comment]
Not really. It's just that I like to have everything in classes. In this case it is not necessary (yet), so replacing the class and method with a function is surely enough.
